scripts/update-gradle-compatibility.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/sh
+
+# Script that update files required for the Android Gradle Plugin version compatibility with
```

Comment:
Switching gradle plugin requires some changes to compile, so I moved them to script so it is easier to maintain.